stable/prometheus-operator/templates/prometheus/rules/k8s.rules.yaml

```diff
@@ -22,39 +22,25 @@ spec:
     rules:
```

Comment:
Did you intend to include a rules change in this PR? I re-ran the dashboards and rules sync scripts and they look like they're pulling in quite  a lot more changes than this currently.

There have been issues syncing rules and dashboards in the past, so I think it's worth going through them in detail when they're done. Do you want to make a separate PR for that or do you want to re-run the full sync scripts and validate the result?

Reply:
I'll undo this change then... its safer I guess :)